src/vendor/README.md

```diff
@@ -1,168 +1,28 @@
-# Highlight.js
+# Highlight.js CDN Assets
```

Comment:
These files are not useful. I removed them and cleaned up the `vendor` directory in https://github.com/ocaml/odoc/pull/915

Reply:
OK thanks - does that PR also contain the updated highlight.js?

Reply:
No, it's the old one but I can cherry pick your commit.

Reply:
I've cherry-picked your commit about updating highlightjs into https://github.com/ocaml/odoc/pull/915 and rebased it on top of the new support files directory.